lib/TestExecutor.py

```diff
@@ -397,6 +402,58 @@ def launch_case(self, obj_xmlnode_next_case):
 
         return
 
+    def check_infrasim_version(self, obj_case):
+        obj_case.stack = Env.obj_stack
+        for obj_rack in obj_case.stack.walk_rack():	
+            list_node = obj_rack.walk_node()
+            infrasim_version = ""
+            infrasim_version_ls = []
+            infrasim_compute_latest_commit = ""
+            infrasim_compute_latest_commit_ls =[]
+            for obj_node in list_node:
+                node_name = obj_node.get_name()
+                node_ip =  obj_node.get_ip()
+                node_username = obj_node.get_username()
+                node_password = obj_node.get_password()
+               
+		 # OpeN ssh client
```

Comment:
Why the "N" in "OpeN" capitalized? Pls follow comment convention. So is line 445. And there is a typo in 424, 
chech ->check?